src/main/java/uk/gov/beis/els/api/openapi/OpenApiConfig.java

```diff
@@ -20,7 +20,7 @@ public OpenAPI configureOpenApi() {
         new Info()
             .title("Create an energy label")
             .description("Create a label for an energy-related product (ErP) that you make or sell to UK customers, for example white goods, boilers and other household appliances.")
-            .version("v1")
+            .version("v1.1")
```

Comment:
In the 'versioning' section of the doco we say 'We'll only change the version number of the API if we make changes that might break applications using the API.'

Perhaps we should update that section to say `major version` where applicable because I agree that point releases are useful for new non-breaking features